src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectState.cs

```diff
@@ -238,7 +226,7 @@ public ProjectState WithAddedHostDocument(HostDocument hostDocument, Func<Task<T
                 }
             }
 
-            var state = new ProjectState(this, ProjectDifference.DocumentAdded, HostProject, WorkspaceProject, documents, importsToRelatedDocuments);
+            var state = new ProjectState(this, ProjectDifference.DocumentAdded, HostProject, TagHelpers, documents, importsToRelatedDocuments);
```

Comment:
Can this be another object rather than the tag helpers? there's gon be more stuff.